src/Components/Blazor/Blazor/src/Http/WebAssemblyHttpMessageHandler.cs

```diff
@@ -25,10 +25,10 @@ public class WebAssemblyHttpMessageHandler : HttpMessageHandler
         public static FetchCredentialsOption DefaultCredentials { get; set; }
             = FetchCredentialsOption.SameOrigin;
 
-        static object _idLock = new object();
-        static int _nextRequestId = 0;
-        static IDictionary<int, TaskCompletionSource<HttpResponseMessage>> _pendingRequests
+        private static readonly object _idLock = new object();
+        private static readonly IDictionary<int, TaskCompletionSource<HttpResponseMessage>> _pendingRequests
```

Comment:
Do we need `IDictionary<>` here? This is strictly worse than using `Dictionary` directly (inlining and such).

Reply:
I think this one was my fault. I used to do this everywhere.